src/simplewallet/simplewallet.cpp

```diff
@@ -3256,8 +3362,8 @@ bool simple_wallet::get_tx_key(const std::vector<std::string> &args_)
 //----------------------------------------------------------------------------------------------------
 bool simple_wallet::get_tx_proof(const std::vector<std::string> &args)
 {
-  if(args.size() != 2 && args.size() != 3) {
-    fail_msg_writer() << tr("usage: get_tx_proof <txid> <dest_address> [<tx_key>]");
+  if(args.size() != 2) {
+    fail_msg_writer() << tr("usage: get_tx_proof <txid> <dest_address>");
```

Comment:
I propose to drop this optional syntax, so that the `get_tx_proof` command works straightforwardly with the additional tx keys; otherwise, we'd have to deal with the standard tx key and the additional tx keys with the `get_tx_key` command, which seems complicated to me. I think the direct display of the tx secret key via `get_tx_key` can be retired in favor of `get_tx_proof` as the safer alternative.